### DISCUSSION TITLE
Adding a specific image to another?

### DISCUSSION THREAD

**Discussion by inar-vision** [lllyasviel/Fooocus#1746] [Q&A]
![tested](https://github.com/lllyasviel/Fooocus/assets/64497640/f210f53b-3379-4abf-b6d1-18818fb560cb)
So what if I have a portrait of a person and a product photo of some product. How should I configure the settings, if I'd like the portrait-person to pose with that specific product in some setting?

As in this image, I'm able to make her hold some bottle of shampoo, but not that specific bottle..

**Comment by wisdombox44** [comment]
This may help
https://www.youtube.com/watch?v=-LMJYw4nYHo
It is good idea to use some image editor for composition like photoshop, gimp, photopea and render the image again

**Reply by inar-vision** [reply]
Thanks. I thought this might be the process, but just tried out if someone knew a way to prompt this out.. Thanks anyways, I'll start photoshopping things.

**Comment by rucko24** [comment]
I'm doing exactly the same thing. I need a high-resolution input image, and then in Outpaint I highlight the area where I want my input object...

But I see that with some design tools you have to do it first and then render it, as my fellow user @wisdombox44 says.

**Comment by screwballl** [comment]
Fooocus is not designed for this process. There are other AI image generators that could but they tend to need much better graphics cards to do so at decent speeds.
